internal/services/contracts_events_consumer.go

```diff
@@ -178,11 +184,11 @@ func (c *ContractsEventsConsumer) handleAfterMarketTransferEvent(e *ContractEven
 
 	apUnit.UserID = null.String{}
 	apUnit.OwnerAddress = null.BytesFrom(args.To.Bytes())
+	apUnit.Beneficiary = null.Bytes{}
 
 	cols := models.AutopiUnitColumns
 
-	_, err = apUnit.Update(ctx, c.db.DBS().Writer, boil.Whitelist(cols.UserID, cols.OwnerAddress))
-	if err != nil {
+	if _, err = apUnit.Update(ctx, c.db.DBS().Writer, boil.Whitelist(cols.UserID, cols.OwnerAddress, cols.Beneficiary)); err != nil {
```

Comment:
Could you do us a small favor and include `updated_at` here too? I know this was a problem before you got her ebut

Reply:
I'll just do it.